great-idea/index.html

```diff
@@ -17,7 +17,89 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+<div class="site-wrapper">
+<header>
+
+    <nav class="navbar">
+      <a href="services.html">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+      
+      <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
       
+      </nav>
+</header>
+ 
+
+<section class="home">
+  <div class="home-child1">
+    <h1>Innovation</h1>
+        <h1>
+            On
+        </h1>
+        <h1>Demand</h1> 
+    <button>Get Started</button>
+  </div>
+    
+  <div class="home-child">
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+  </div>
     
+</section>
+
+<section class="main">
+
+<div class="top-content">
+  <div class="text-top">
+    <h2>Features</h2>
+  
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </div>
+
+  <div class="text-top">
+    <h2>About</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </div>
+</div>
+
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+<div class="bottom-content">
+    <div class="text-bottom">
+        <h2>Services</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+       </div> 
+      
+      <div class="text-bottom">
+        <h2>Product</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+      
+      <div class="text-bottom">
+        <h2>Vision</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+</div>  
+</section>
+
+<div class="contact">
+      <h2>Contact</h2>
+    <address>
+        <p>123 Way 456 Street</p>
+            <p>Somewhere, USA</p>
```

Comment:
spacing here as well.  It is important for readability later.  When needing to identify parent/child elements indentation will make a difference.